### PR TITLE
Data flow

### PR DESCRIPTION
**Pull request by joverlee521**
Documentation of data flow within SFS & SCAN.

### CODE REVIEW

**Comment by joverlee521** [comment]
[Document the various study arms and how the data are imported](https://trello.com/c/hz9Z4tKL/415-document-the-various-study-arms-and-how-the-data-are-imported)

File: data-flow.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+##### Cepheid
+* Part of the Kiosk study arm that provides rapid diagnostics on-site
+* Kiosk staff manually enters the results in the Kiosk REDCap project
+* Provides results for Influenza A, Influenza B, and RSV.
+
+#### Genomic Data
+##### NWGC
+* Uses Illumina sequencing to sequence samples that have tested postive for certain pathogens.
+* NWGC partners upload the resulting raw sequence reads to Globus.
+* Sequence reads are manually transferred from Globus to Fred Hutch rhino
```
Comment (kairstenfay) [inline data-flow.md:103]: What is "Fred Hutch rhino"? The new developers at BBI may not understand.

File: data-flow.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+* Requires manual parse and upload of metadata for these sequence read sets
+    * see detailed script in [ID3C](https://github.com/seattleflu/id3c/blob/master/lib/id3c/cli/command/sequence_read_set.py)
+
+##### SFS Assembly
+* The sequence reads are assembled into consensus genomes using the SFS [assembly] pipeline
+* Currently only assembling Influenza and SARS-CoV-2 positive samples.
+* The SFS assembly pipeline automatically POSTs complete Influenza genomes to ID3C table `receiving.consensus_genome`
```
Comment (kairstenfay) [inline data-flow.md:110]: Via which API endpoint?

File: data-flow.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+[ITHS] then helps us pull data from EMR and fills in the records on REDCap.
+* See detailed script in [backoffice](https://github.com/seattleflu/backoffice/blob/master/bin/import-uw-retrospectives-to-redcap)
+
+#### REDCap
+In Year 2 of SFS, we started to use the [ITHS REDCap] survey tools to collect data from participants.
+
+Each study arm sets up their own REDCap project so they can customize the questions and flow of the survey according
+to the needs of the study. Due these differences, we've set up separate ingest processes for all of the
+SFS projects. We use a single ingest process for all SCAN projects since they have identical survey fields, just
+translated into different languages.
```
[the remark this answers is not in the document]
Comment (joverlee521) [reply data-flow.md:39]: Yeah, in the line before, I mentioned that we've set up separate ingest processes for all of the SFS projects. I can make the language more clear here to contrast the two.

File: data-flow.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+
+## Data Ingest
+
+#### Lab Manifest
+The lab at the Brotman Baty Institute (BBI) maintains Excel files that record all samples processed by them.
+* Uploaded daily to the Fred Hutch AWS S3 bucket by BBI partners
+* Cronjobs set up to parse and upload the manifest data to ID3C table `receiving.manifest`
+* See detailed script in [specimen-manifest] and [ID3C](https://github.com/seattleflu/id3c/blob/master/lib/id3c/cli/command/manifest.py).
+
+The lab manifest is also the source from which we identify UW retrospective samples that need metadata pulled EMR.
```
[the remark this answers is not in the document]
Comment (joverlee521) [reply data-flow.md:28]: Ah, yeah, I'll fix the language here

File: data-flow.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+    * [Metabase](#metabase)
+    * [Shipping Schema](#shipping-schema)
+
+
+## Data Ingest
+
+#### Lab Manifest
+The lab at the Brotman Baty Institute (BBI) maintains Excel files that record all samples processed by them.
+* Uploaded daily to the Fred Hutch AWS S3 bucket by BBI partners
+* Cronjobs set up to parse and upload the manifest data to ID3C table `receiving.manifest`
```
[the remark this answers is not in the document]
Comment (joverlee521) [reply data-flow.md:25]: Yeah, I can point to the `id3c-production` crontab file

File: data-flow.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kairstenfay) [reply data-flow.md]: Oh I probably read it too quickly, then. Sorry about that.

**Pull request merged**